mapper/pojo-standalone/src/main/java/org/hibernate/search/mapper/pojo/standalone/massindexing/MassIndexer.java

```diff
@@ -143,6 +143,19 @@ public interface MassIndexer {
 	 */
 	MassIndexer failureHandler(MassIndexingFailureHandler failureHandler);
 
+	/**
+	 * Sets the maximum number of failures that will be reported and sent to {@link MassIndexingFailureHandler} per indexing operation.
+	 * Any failures exceeding this number will be ignored. A count of such ignored failures together with the operation they belong to
+	 * will be logged upon the completion of indexing process.
+	 * <p>
+	 * By default, this number is set to {@code 1000}.
+	 *
+	 * @param maximumFailuresHandledPerOperation The number representing the maximum failures to be handled per operation.
+	 * @return {@code this} for method chaining
+	 */
+	@Incubating
+	MassIndexer maximumFailuresHandledPerOperation(long maximumFailuresHandledPerOperation);
```

Comment:
I would stay vague in the method name, because it's one of those cases where adding more info is just confusing. Maybe `failureLimit`? `failureFloodingThreshold`?